package.json

```diff
@@ -23,6 +23,7 @@
     "typecheck": "tsc",
     "audit-prod": "npx better-npm-audit audit --production -x 1091470",
     "deploy": "sls deploy",
+    "destroy": "sls destory",
```

Comment:
looks like a typo here

Reply:
lol, I re-added this because I kept having to add it locally, but I hadn't actually run it this time to tear down my test deploy, so of course I'd typo it. fixed.